ios/FluentUI/DrawerVnext/SlideOverPanel.swift

```diff
@@ -115,26 +149,7 @@ struct SlideOverPanel<Content: View>: View {
                     .onTapGesture (perform: actionOnBackgroundTap ?? {})
             }
         }
-        .background(transitionState == .collapsed ? tokens.backgroundClearColor : tokens.backgroundDimmedColor.opacity(resolveTransitionOpacity(tokens.backgroundDimmedOpacity)))
-    }
-
-    private func resolveTransitionOpacity(_ maxValue: Double) -> Double {
-        let clearBackgroundOpacity = 0.0
-        guard backgroundDimmed else {
-            return clearBackgroundOpacity
-        }
-
-        switch transitionState {
-        case .collapsed:
-            return clearBackgroundOpacity
-        case .expanded:
-            return maxValue
-        case .inTransisiton:
-            if let percent = percentTransition {
-                return maxValue * percent
-            }
-        }
-        return 0
+        .background(transitionState == .collapsed ? tokens.backgroundClearColor : tokens.backgroundDimmedColor.opacity(resolvedBackgroundOpacity))
```

Comment:
This logic could be moved to the **backgroundColor** computed property:

`transitionState == .collapsed ? tokens.backgroundClearColor : tokens.backgroundDimmedColor.opacity(resolvedBackgroundOpacity)`